stockflux-securities-master/src/components/Button.css

```diff
@@ -0,0 +1,42 @@
+.button-container.extraSmall > button,
+.button-container.small > button,
+.button-container.large > button {
```

Comment:
You've got a lot of repeated classes here in your CSS. I would recommend having a single base class with a base style and then adding style for modifiers to the class.

```css
.button {
    /*base style here*/
}

.button.small {
  /*All overrides for small here*/
}

.button.small > span {
  /*Any overrides for direct span children of small buttons here*/
}
```

Reply:
Yep, gotten rid of this unnessessary stuff now